types/applications.go

```diff
@@ -86,6 +86,7 @@ type ApplicationCallTxnFields struct {
 	GlobalStateSchema StateSchema `codec:"apgs"`
 	ApprovalProgram   []byte      `codec:"apap"`
 	ClearStateProgram []byte      `codec:"apsu"`
+	ExtraProgramPages uint32      `codec:"apep,omitempty"`
```

Comment:
Why `omitempty` here? I would assume all "optional" fields such as `Accounts` should be `omitempty` or none of them should.

Reply:
That sounds right. I think the "omitempty" on the _struct field covers it.